### PR TITLE
content update

### PR DESCRIPTION
**Pull request by amigos-sian**

### CODE REVIEW

File: docs/install-diego/diego-init-aws.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+Command will request for:
+
+• AWS account ID
+
+• AWS region
+
+• AWS EKS Cluster Name
+
+• AWS EKS Cluster OIDC Provider URL
+
```
Comment (techamigo) [inline docs/install-diego/diego-init-aws.md:21]: Provide a one line description for each input parameter

**Review by techamigo — COMMENTED** [review]
Here are some comments. Feel fre to
Make those changes as you see appropriate and merge

File: docs/how-does-diego-work/setup-for-open-source-tools.md [hunk 2]
```diff
@@ -19,7 +19,7 @@ Here are some tools used at a high level:
 
 Here are open source tools deployed as part **Diego-Core** in your Kubernetes cluster as part of a Helm Chart:
 
-- [argocd](https://argo-cd.readthedocs.io/en/stable/)
+- [Argo CD](https://argo-cd.readthedocs.io/en/stable/)
```
[the remark this answers is not in the document]
Comment (amigos-sian) [reply docs/how-does-diego-work/setup-for-open-source-tools.md:22]: these should already be included in the code below each of these. kept it simple and also have included appropriate links for doc pages for context.

File: docs/how-does-diego-work/setup-for-aws.md [hunk 3]
```diff
@@ -26,6 +26,6 @@ The Cloudformation stack provisions:
 
         - Github Actions - to allow CI pipelines to access resources in cluster
 
-        - IAM Roles for Server Accounts (IRSA) on EKS - establishes a trust relationship between specific external identity provider (IdP) services that support OpenID Connect (OIDC) and your organisation’s AWS account. More information here.  
+        - IAM Roles for Server Accounts (IRSA) on EKS - establishes a trust relationship between specific external identity provider (IdP) services that support OpenID Connect (OIDC) and your organisation’s AWS account. [More information here](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_providers_create_oidc.html) 
 
 - The 3 key components of Diego are installed into your AWS EKS cluster, alongside a toolchain of open source tooling which make up Diego-Core.
```
[the remark this answers is not in the document]
Comment (amigos-sian) [reply docs/how-does-diego-work/setup-for-aws.md:31]: cool, np

File: docs/how-does-diego-work/setup-for-github.md [hunk 4]
```diff
@@ -6,5 +6,5 @@ sidebar_position: 3
 
 ## GitHub setup
 
-- Diego installs a Github app for your organisation which sets up permissions to help implement GitOps between repositories for your applications and ArgoCD.
-- Diego sets up template repos in your organisation, required for Diego. These provide configuration files for applications and supporting tooling, as part of a GitOps model.
\ No newline at end of file
+• Diego installs a Github app for your organisation which sets up permissions to help implement GitOps between repositories for your applications and ArgoCD.
```
[the remark this answers is not in the document]
Comment (amigos-sian) [reply docs/how-does-diego-work/setup-for-github.md:9]: ok

File: docs/install-diego/diego-init-aws.md (hunk 1 above)
Comment (amigos-sian) [reply docs/install-diego/diego-init-aws.md:21]: underway...

**Pull request merged**